packages/autorest.python/autorest/codegen/templates/vendor.py.jinja2

```diff
@@ -71,11 +71,16 @@ class NamedBytesIO(BytesIO):
         super().__init__(*args, **kwargs)
         self.name = name
 
-def multipart_form_data_file(file: Union[IOBase, bytes]) -> IOBase:
+def multipart_file(file: Union[IOBase, bytes]) -> IOBase:
     if isinstance(file, IOBase):
         return file
     return NamedBytesIO("auto-name-" + str(uuid.uuid4()), file)
 
+def multipart_data(data: Any) -> Any:
+    if isinstance(data, (list, tuple, dict, Model)):
```

Comment:
Since we do not have a good way to determine the content type for each of form part, I'm ok with current implementation. But we shall open a thread to discuss with TSP team about it.